### ISSUE TITLE
You're already passing user into the locals object

### ISSUE THREAD

**Issue by KhalilKhunji** [AyaEhab7/men-stack-relating-data-lab-cookbook#4]
https://github.com/AyaEhab7/men-stack-relating-data-lab-cookbook/blob/b4f007829cf434a86fab8abc4d9d3f7f9043b961/controllers/foods.js#L11-L14
Several times throughout the food controller, you're adding user to the render object, but you already should have access to it in your EJS templates via the locals object, since you're running this middleware in your server.js:

https://github.com/AyaEhab7/men-stack-relating-data-lab-cookbook/blob/b4f007829cf434a86fab8abc4d9d3f7f9043b961/server.js#L42

In other words, adding the user every time you render is redundant, as you already should have access to it through the locals object.
